appnav/src/main/kotlin/io/element/android/appnav/room/joined/JoinedRoomLoadedFlowNode.kt

```diff
@@ -95,6 +98,7 @@ class JoinedRoomLoadedFlowNode @AssistedInject constructor(
             },
             onDestroy = {
                 Timber.v("OnDestroy")
+                activeRoomHolder.removeRoom(inputs.room.sessionId)
```

Comment:
This does not work well with permalink. If a room A opens a room B, when going back to room A, the holder will be empty.